dev/sc.R

```diff
@@ -0,0 +1,71 @@
+# SC (ophthalmology)
```

Comment:
Could we reference the pkgdown site - just for future devs coming to look at this script and wondering what it is for?

Reply:
there is work being done on the site, for the upcoming release. did we do this for `onco`, should we maybe have a page with the different admiral package sites where `admiral.test` is being used?

Reply:
not a standard practice - but I would like to start pointing to sites/repos of where this data is being used.  Just to let people a few years down the road know about where the data is being utilized.

If not available - maybe we could just point to the repo.

Reply:
We could put a blurb in the news that admiral.test has sdtm data for oncology and opthat available and link to pkgdown sites.  I'll make an issue to do this before the release.